test/ordered_hierarchy_test.clj

```diff
@@ -24,14 +28,14 @@
     (is (nil? (parents h ::text)))
     (is (= [::text] (vec (parents h ::varchar-255))))
     (is (= [::float] (vec (parents h ::int))))
-    (is (= [::boolean ::int] (vec (parents h ::boolean-or-int))))))
+    (is (= [::boolean ::explicit-int] (vec (parents h ::boolean-or-int))))))
```

Comment:
I get that ordered-hierarchy tests are orthogonal to CSV type-detection testing, but it doesn't seem right to shadow the type hierarchy. I would either:

1.  [preferred] come up with a new hierarchy just for testing. Polygons are a classic example (especially with the quadrilateral/rhombus/parallelogram/rectangle/square/trapezoid soup), or you could do animal species or musical instruments or modes of transportation or whatever your heart desires.

2. Recycle the hierarchy from CSV uploading.